lib/saml/templates/sso_post_form.html.erb

```diff
@@ -26,7 +26,15 @@
 
   <script>
     (function() {
-      document.getElementById("saml-form").submit();
+      var form = document.getElementById("saml-form");
+      var req = new XMLHttpRequest();
+      var qs = "saml_uuid=" + encodeURIComponent("<%= saml_uuid %>") +
+                "&authn=" + encodeURIComponent("<%= authn %>") +
+                "&type=" + encodeURIComponent("<%= type %>");
+      var url = location.origin + "/v1/sessions/tracker?" + qs;
+      req.open("GET", url);
+      req.send();
```

Comment:
If a user is having trouble with the form submit in their browser, what are the chances that this JS won't work as expected either?

Reply:
@jholton do you mean how often do I think the XHR request will fire but the form submission will fail?

Reply:
Yeah, that's what I meant. 

I guess my question is: how does this get us insight into if/which users are having trouble submitting the form?
Is it comparing logging of the SAML UUID in [`render_login`](https://github.com/department-of-veterans-affairs/vets-api/blob/bb9a54929eded71c34848f3844994b92fcc9b7b3/app/controllers/v1/sessions_controller.rb#L188) to the logging of SAML UUID in [`tracker`](https://github.com/department-of-veterans-affairs/vets-api/blob/bb9a54929eded71c34848f3844994b92fcc9b7b3/app/controllers/v1/sessions_controller.rb#L78)?

Should/can we log user agent info too to determine which browsers are having trouble?

Reply:
Right now we are assuming every `STATSD_SSO_SAMLREQUEST_KEY` issued is a request that should be showing up at eauth.va.gov.  However if there are any issues with this form loading, not neccessarlily submitting, the XHR request will fail to fire and the subsequent `STATSD_SSO_SAMLTRACKER_KEY` won't increment.

Basically if the `STATSD_SSO_SAMLREQUEST_KEY` doesn't match `STATSD_SSO_SAMLTRACKER_KEY` then we have an issue with the form loading, if they do match it just confirms that the problem is most likely downstream of us.